### DISCUSSION TITLE
Domanda circa raccomandazioni su installer Noobs e alimentatore su Raspberry Pi 4B

### DISCUSSION THREAD

**Discussion by xdiego76** [indomus/forum#2368] [HUB: Home Assistant]
Buongiorno a tutti,
a fine 2019 avevo preso un kit con un Raspberry Pi 4B 4GB RAM. Rileggendo una vecchia mail risalente al tempo dell'acquisto, mi son accorto che mi avevano fatto delle raccomandazioni circa la versione di Noobs (il software di installazione da utilizzare per scegliere il sistema operativo) e la compatibilità legata all'alimentatore in dotazione (nel kit c'era un 5.1V 3A).
Mi piacerebbe sapere cosa ne pensate, se c'è qualcosa di vero (perché mi pare che NOOBS non si usi più), e se il discorso dell'alimentatore ha senso o meno.

Ecco il testo:

> Salve,
> 
> Questa è Livia da TrekPow IT. Grazie per aver acquistato Raspberry Pi 4B kit nel nostro negozio di Amazon.
> 
> Vogliamo dare dei consigli su uso per questo kit.
> Alimentatore 5.1V:
> 
> L’alimentatore nel pacchetto è sufficiente per il board di Raspberry Pi.  Se si sta utilizzando un sistema operativo diverso da quello originariamente pre-caricato sul microSD e si vede un simbolo di un fulmine sullo schermo, indica che si sta utilizzando un sistema operativo NON compatibile. NON è correlato alla tensione di alimentazione.
> Il modello "4B" richiede il software compatibile "4B " per funzionare, come ad esempio il NOOBS originale programmato 2.7.0 o successivo. Assicurati di utilizzare un sistema operativo compatibile o la scheda non si avvierà. Si prega di notare che il pi 4 supporta il sistema NOOBS 7.0, 7.1 o 7.2. E deve installare il sistema da microSD slot!
> 
> Si prega di notare che il Raspberry Pi 4B non supporta il sistema di recalbox18.04.20, lakka, OpenELEC, se c'è un'icona di un piccolo fulmine nell'angolo, si prega di confermare anche il sistema installato.
> 

Grazie!
Diego

**Comment by KC-inDomus** [comment]
è una scemenza

**Reply by xdiego76** [reply]
Ottimo, come immaginavo! :-D
